config.yaml

```diff
@@ -7,7 +7,11 @@ logging:
 
 plugins:
   backbones:
-    inmem:
+    #inmem:
```

Comment:
Why comment this one?

Reply:
Because everything would be provisioned twice otherwise.